sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -114,14 +114,27 @@ analyze sentiment, recognize entities, detect language, and extract key phrases
 To create a client object, you will need the cognitive services or text analytics endpoint to 
 your resource and a subscription key that allows you access:
 
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L45-L48 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L44-L47 -->
 ```java
 TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
-    .subscriptionKey(SUBSCRIPTION_KEY)
-    .endpoint(ENDPOINT)
+    .subscriptionKey(new TextAnalyticsSubscriptionKeyCredential("{subscription_key}"))
+    .endpoint("{endpoint}")
     .buildClient();
 ```
 
+#### Rotate existing subscription key
+The Azure Text Analytics client library provide a way to rotate the existing subscription key.
+
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L174-L180 -->
+```java
+TextAnalyticsSubscriptionKeyCredential credential = new TextAnalyticsSubscriptionKeyCredential("{expired_subscription_key}");
```

Comment:
I wouldn't call it `expired_subscription_key` as it might give the wrong message to people when creating the client.
it could be just `subscription_key`, and leave `new_subscription_key`